sources/packages/forms/src/form-definitions/educationprogram.json

```diff
@@ -1,5871 +1,6243 @@
-      "tree": false,
-      "lazyLoad": false,
-      "id": "e1fg1xb"
-    },
-    {
-      "label": "HTML",
-      "tag": "p",
-      "className": "",
-      "attrs": [
-        {
-          "attr": "",
-          "value": ""
-        }
-      ],
-      "content": "Entrance requirements",
-      "refreshOnChange": false,
-      "customClass": "category-header-medium primary-color",
-      "hidden": false,
-      "modalEdit": false,
-      "key": "html8",
-      "tags": [],
-      "properties": {},
-      "conditional": {
-        "show": null,
-        "when": null,
-        "eq": "",
-        "json": ""
-      },
-      "customConditional": "",
-      "logic": [],
-      "attributes": {},
-      "overlay": {
-        "style": "",
-        "page": "",
-        "left": "",
-        "top": "",
-        "width": "",
-        "height": ""
-      },
-      "type": "htmlelement",
-      "input": false,
-      "placeholder": "",
-      "prefix": "",
-      "suffix": "",
-      "multiple": false,
-      "defaultValue": null,
-      "protected": false,
-      "unique": false,
-      "persistent": false,
-      "clearOnHide": true,
-      "refreshOn": "",
-      "redrawOn": "",
-      "tableView": false,
-      "dataGridLabel": false,
-      "labelPosition": "top",
-      "description": "",
-      "errorLabel": "",
-      "tooltip": "",
-      "hideLabel": false,
-      "tabindex": "",
-      "disabled": false,
-      "autofocus": false,
-      "dbIndex": false,
-      "customDefaultValue": "",
-      "calculateValue": "",
-      "calculateServer": false,
-      "widget": null,
-      "validateOn": "change",
-      "validate": {
-        "required": false,
-        "custom": "",
-        "customPrivate": false,
-        "strictDateValidation": false,
-        "multiple": false,
-        "unique": false
-      },
-      "allowCalculateOverride": false,
-      "encrypted": false,
-      "showCharCount": false,
-      "showWordCount": false,
-      "allowMultipleMasks": false,
-      "id": "e1z1fry",
-      "addons": []
-    },
-    {
-      "title": "Panel",
-      "labelWidth": "",
-      "labelMargin": "",
-      "theme": "default",
-      "tooltip": "",
-      "customClass": "",
-      "collapsible": false,
-      "hidden": false,
-      "hideLabel": true,
-      "disabled": false,
-      "modalEdit": false,
-      "key": "panel8",
-      "tags": [],
-      "properties": {},
-      "customConditional": "",
-      "conditional": {
-        "json": "",
-        "show": null,
-        "when": null,
-        "eq": ""
-      },
-      "logic": [
-        {
-          "name": "Disable Entrance Inputs",
-          "trigger": {
-            "type": "javascript",
-            "javascript": "result = data.hasOfferings;"
-          },
-          "actions": [
-            {
-              "name": "Disable Inputs",
-              "type": "property",
-              "property": {
-                "label": "Disabled",
-                "value": "disabled",
-       
```

Comment:
label and key doesnot match

Reply:
As explained in our chat, I did not make any update on this part of the form.